server/routes/star.js

```diff
@@ -0,0 +1,98 @@
+import { Router } from "express";
+import pool from '../db.js'
+
+const router = Router();
+
+
+router.get("/", async (req, res) => {
+   
+    try {
+        const stars = await pool.query("SELECT * FROM stars");
+        res.status(200).json(stars.rows)
```

Comment:
+1 of your use of response codes, this is good API design. It might be good to think if the number of rows === 0 then should we return a 404?

Reply:
I'm thinking frontend still needs an empty array even there is no item in star table. so in this case still the status is successful so I prefer to keep 200 for this senario and pass either empty array or array with stars in it .
Btw, there is another API which returns one star base on star id param and I'm going to implement 404 for that API  beacuse user is looking for a specific star and if it didnt find then we need to tell them we couldnt find it for you.
what do you think about this approach?

Reply:
This approach sounds good to me